substrate/frame/system/src/lib.rs

```diff
@@ -448,6 +461,14 @@ pub mod pallet {
 
 		/// The maximum number of consumers allowed on a single account.
 		type MaxConsumers: ConsumerLimits;
+
+		type SingleBlockMigrations: OnRuntimeUpgrade;
+
+		type MultiBlockMigrator: MultiStepMigrator;
```

Comment:
Yea indeed. Currently i wanted to have a grace period where it is possible to configure the migrations on either `Executive` or `System`. But maybe we can do it in one since it is already a breaking change.